app/assets/javascripts/discourse/app/templates/components/invite-link-panel.hbs

```diff
@@ -0,0 +1,67 @@
+{{#if inviteModel.error}}
+  <div class="alert alert-error">
+    {{html-safe errorMessage}}
+  </div>
+{{/if}}
+
+<div class="body">
+  {{#if inviteModel.finished}}
+    {{generated-invite-link link=inviteModel.inviteLink email=null}}
```

Comment:
do we need email=null here ? why not just removed the email property

Reply:
Good idea. Made it so.